README.md

```diff
@@ -93,14 +93,6 @@ There is a number of dependencies needed to make kubefarm working:
   kubectl create secret generic -n metallb-system memberlist --from-literal=secretkey="$(openssl rand -base64 128)"
   ```
 
-  There is currently a bug in MetalLB that may block the use of multiple services on shared IP [metallb/metallb#558](https://github.com/metallb/metallb/issues/558).
-  However you can simple use patched images from this PR [metallb/metallb#562 (comment)](https://github.com/metallb/metallb/pull/562#issuecomment-724066537):
-
-  ```bash
-  kubectl set image -n metallb-system deploy/controller controller=docker.io/kvaps/metallb-controller:a3047c4d
-  kubectl set image -n metallb-system ds/speaker speaker=docker.io/kvaps/metallb-controller:a3047c4d
-  ```
-
```

Comment:
Hi, thanks for your contribution!

It seems this bug is still not resolved, both issue and PR are have open status.
Don't you experience this kind of problems?

Reply:
Np! 
My mistake, I thought that this just meant 2 services couldn't share the same IP. Wasn't aware of arp conflicts between nodes (I am only running one).

I will undo this commit.

Reply:
I should mention that with my single node cluster I was running into issues with it announcing at all. But this is unrelated.